extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -479,6 +499,11 @@ export class AmpSlideScroll extends BaseSlides {
     this.slideIndex_ = newIndex;
     this.hideRestOfTheSlides_(showIndexArr);
     this.setControlsState();
+
+    if (opt_isUserAction) {
```

Comment:
The idea is I only want to trigger an action if `showSlide_()` is invoked as a result of a user gesture (e.g. tap/swipe rather than during initialization). IMO `isBindAction` is not a good name here since it's not actually caused by Bind.